docs/latency-limitations/latency-limitations.md

```diff
@@ -0,0 +1,37 @@
+# Latency-imposed limitations
+
```

Comment:
I propose to say, that the general rule of thumb is that every 100 km of distance adds roughly ~1 ms of latency.